packages/web/src/app/keywords.css

```
@@ -38,6 +38,39 @@
+
+  position: relative;
+  display: inline-flex;
+  vertical-align: middle;
+  isolation: isolate;
+  font-size: 0.9em;
+}
+
+.card-keyword--tag {
+  --keyword-bg: #0a0a0a;
+  --keyword-fg: #fff;
+  font-size: 1em;
```

Stylelint (17.14.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)